src/oaklib/implementations/pronto/pronto_implementation.py

```diff
@@ -520,6 +520,9 @@ def _clean(v: str) -> str:
                 if syn.description == patch.old_value:
                     syn.description = patch.new_value
         elif isinstance(patch, kgcl.NewSynonym):
+            t = self._entity(patch.about_node, strict=True)
```

Comment:
This line was already present before. For some reason the reverting of #215 messed things up. The only reason I have this PR is to bring the change in lines 526 and 527 to Chris' attention.